mkosi/__init__.py

```diff
@@ -3726,7 +3726,7 @@ def insert_partition(
 
     run(["sfdisk", "--color=never", loopdev], input='\n'.join(table).encode("utf-8"))
     run(["sync"])
-    run(["partx", "--update", loopdev])
+    run(["partprobe", loopdev])
```

Comment:
hmm, this adds a dep onto "parted" which we so far didn't have.

Maybe use "blockdev --rereadpt"?

Reply:
Oh, indeed. For some reason I thought it's from util-linux too. I'll change this.